.travis.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+# Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
We shouldn't change the year. This should be the year when the file was created.